manifests/init.pp

```diff
@@ -49,13 +49,13 @@
       $command = "${utils_cmd} --init-token --slot ${idx} --pin ${token[1]['pin']} --so-pin ${token[1]['so_pin']} --label ${token[0]}"
     } else {
       $pattern = "^\s+Label:\s+${token[0]}\s+$"
-      $command = "${utils_cmd} --init-token --free --pin ${token[1]['pin']} --so-pin ${token[1]['so_pin']} --label ${token[0]}"
+      $command = "${utils_cmd} --init-token --free --pin ${token[1]['pin']} --so-pin ${token[1]['so_pin']} --label ${token[0]} && chown -R ${user}:${group} ${tokendir}"
```

Comment:
this shouldn'ty bee needed because of https://github.com/sacr0/puppet-softhsm/blob/6734f7041785ceb49afacb95ecca77385d66e377/manifests/init.pp#L33-L38.  dose the exec need to be run with the exec with `user => $user` and  `$group => $group`